tests/test_enhanced_error_classification_simplified.py

```
@@ -0,0 +1,355 @@
+            line=1, column=1,
+            rule_id="E901",
+            message="SyntaxError: invalid syntax",
+            severity=ErrorSeverity.ERROR,
+            linter="flake8"
+        )
+        
+        category = self.error_analyzer._categorize_error(syntax_error)
+        priority = self.error_analyzer._calculate_priority(syntax_error, category)
+        
+        # Syntax errors should have high priority (higher number = higher priority)
+        assert priority >= 80
```

Ruff (0.12.2)

Use of `assert` detected

(S101)